src/api/app/components/fuzzy_time_component.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class FuzzyTimeComponent < ApplicationComponent
+  attr_reader :time
+
+  def initialize(time:)
+    super
+    @time = time.utc
+  end
+
+  def human_time_ago
```

Comment:
This method could be renamed to `call` and be an [inline view component](https://viewcomponent.org/guide/templates.html#inline) instead. Having a template for a single `<span>` element isn't needed.

Reply:
This comment still stands, could you address this? Thank you :slightly_smiling_face: